x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/screens/detail/settings_panel.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiTitle, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+import { EuiSpacer } from '@elastic/eui';
+import { useGetPackageInstallStatus } from '../../hooks';
+import { InstallStatus, PackageInfo } from '../../../../types';
+import { InstallationButton } from './installation_button';
+import { useGetDatasources } from '../../../../hooks';
+
+export const SettingsPanel = (
+  props: Pick<PackageInfo, 'assets' | 'name' | 'title' | 'version'>
+) => {
+  const getPackageInstallStatus = useGetPackageInstallStatus();
+  const { data: datasourcesData } = useGetDatasources({
+    perPage: 0,
+    page: 1,
+    kuery: `datasources.package.name:${props.name}`,
+  });
+  const { name, title } = props;
+  const packageInstallStatus = getPackageInstallStatus(name);
```

Comment:
Q. Does Delete apply to Default installed packages? those defined via `x-pack/plugins/ingest_manager/common/types/models/epm.ts:252`: 

```typescript
export enum DefaultPackages {
  base = 'base',
  system = 'system',
  endpoint = 'endpoint',
}
```

My concern is the impact to Endpoint if uninstalled.
cc:/ @jonathan-buttner , @kevinlog

Reply:
@paul-tavares they are _default_ installed vs _required_ or _always_ installed, IMO. Perhaps `base` is always there, but I believe everything else can be removed.

Reply:
@paul-tavares what @jfsiii said.  Base can technically be removed if you hit the endpoint but we do not allow them to remove it through the UI.

Reply:
Do not want to block this PR, but I think not to be able to remove a package should not only be a UI feature but also be blocked through the API as otherwise it would mess up things.